packages/wrangler/src/api/integrations/platform/index.ts

```diff
@@ -85,6 +85,11 @@ export type GetPlatformProxyOptions = {
 	 * If `false` is specified no data is persisted on the filesystem.
 	 */
 	persist?: boolean | { path: string };
+	/**
+	 * Wether only local bindings should be used. This effectively disables the remote aspect of remote bindings
+	 * (exactly as if their `remote` configuration was changed from `true` to `false`)
+	 */
```

Comment:
Could this be shorter? Maybe something like:

```suggestion
	 * Make all bindings local, ignoring the `remote` value in their config
	 */
```

Reply:
Updated to simply
```js
/**
 * Wether remote bindings should be enabled or not (defaults to `true`)
 */
```
🙂